spyder/widgets/fileswitcher.py

```diff
@@ -190,20 +191,20 @@ def recurse_level(level_idx):
 class KeyPressFilter(QObject):
     """Use with `installEventFilter` to get up/down arrow key press signal."""
     UP, DOWN = [-1, 1]  # Step constants
-
     sig_up_key_pressed = Signal()
     sig_down_key_pressed = Signal()
     sig_enter_key_pressed = Signal()
+    sig_mouse_clicked = Signal()
+    itemClicked = Signal()
 
     def eventFilter(self, src, e):
         if e.type() == QEvent.KeyPress:
             if e.key() == Qt.Key_Up:
                 self.sig_up_key_pressed.emit()
             elif e.key() == Qt.Key_Down:
                 self.sig_down_key_pressed.emit()
-            elif (e.key() == Qt.Key_Enter) or (e.key() == Qt.Key_Return):
+            elif (e.key() == Qt.Key_Return):
```

Comment:
@juanis2112 did you tested this on Mac, Linux and Windows?

Reply:
@dalthviz yes, it works for the three of them.